src/BenchmarkDotNet/Engines/CacheClearingStrategies/NativeCacheClearingStrategy.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+namespace BenchmarkDotNet.Engines.CacheClearingStrategies
+{
+    internal class NativeCacheClearingStrategy : ICacheClearingStrategy
+    {
+        [DllImport("kernel32.dll")]
+        private static extern bool FlushInstructionCache(IntPtr hProcess, IntPtr lpBaseAddress, UIntPtr dwSize);
```

Comment:
Am I understand correctly that `FlushInstructionCache` flushes only instruction cache instead of CPU L1/L2/L3 chaches? In this case, we don't need this kind of strategy.
P.S. See https://blogs.msdn.microsoft.com/oldnewthing/20031208-00/?p=41583